src/ActionHandler/NotificationUpdateHandler.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace CommonGateway\VrijBRPToZGWBundle\ActionHandler;
+
+use CommonGateway\VrijBRPToZGWBundle\Service\NewSynchronizationService;
+use CommonGateway\VrijBRPToZGWBundle\Service\VrijBrpService;
+use CommonGateway\CoreBundle\ActionHandler\ActionHandlerInterface;
+
+class NotificationUpdateHandler implements ActionHandlerInterface
+{
+
+
+    public function __construct(
+        private readonly VrijBrpService $vrijBrpService,
+    ) {
+
+    }//end __construct()
+
+
+    /**
+     *  This function returns the requered configuration as a [json-schema](https://json-schema.org/) array.
+     *
+     * @throws array a [json-schema](https://json-schema.org/) that this  action should comply to
+     */
```

Comment:
Missing @return tag in function comment

Reply:
This has been updated, but somehow Github does not show (@MWest2020 )